otterdog/eclipse-tractusx.jsonnet

```diff
@@ -195,7 +195,7 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
-    orgs.newRepo('e2e-testing') {
+    orgs.newRepo('tractus-x-umbrella') {
```

Comment:
in order to rename a repo you also need to specify the old name by adding

`aliases: ['e2e-testing'],`

as otterdog does not keep a state.

Reply:
[687cee7](https://github.com/eclipse-tractusx/.eclipsefdn/pull/59/commits/687cee7de1a2e21fa01939b4838b16d1f2f9703f)